Dockerfile_Ubuntu_2204

```diff
@@ -0,0 +1,127 @@
+FROM ubuntu:22.04
```

Comment:
I think it is better to move all dockerfiles inside `Dockerfiles/` folder.

Reply:
I dont disagree but we have CI tied to that too. Do we want to do this part of this PR or a down the road thing?

Reply:
> Do we want to do this part of this PR or a down the road thing? 

I think it should be easy to modify CI. It should just be about changing paths for `docker build -f Dockerfile.docker`, if so we can do it in this PR.

Reply:
> I think it is better to move all dockerfiles inside Dockerfiles/ folder.

We should have one Dockerfile a the top-level, and other docker files should be moved to `tools/docker`, and docker files(except the top-level Dockerfile) should be named *.docker.

Reply:
Done